app/src/main/java/factorio/train/analyser/App.java

```diff
@@ -3,12 +3,21 @@
  */
 package factorio.train.analyser;
 
+import java.io.UnsupportedEncodingException;
+import java.util.zip.DataFormatException;
+import factorio.train.analyser.Decoder.Decoder;
+
 public class App {
-    public String getGreeting() {
-        return "Hello World!";
+
+    private String blueprint = "0eNqV19tugkAQBuB3mWtInF1gWV6laRoPG7uJogE0NYZ37yLGtOk0/HPpgc+Bmd+BO20Ol3DuYjtQc6e4PbU9NW936uO+XR+m94bbOVBDcQhHyqhdH6dX3ToeaMwotrvwRQ2P7xmFdohDDPPxjxe3j/Zy3IQufeF15PbSXcMufwAZnU99OubUTj+UHJPRjZqcTaJ3sQvb+bNqzP6IBhN5JZNWIO2L7Iek7T+Hf9CcZ9T/Np1gFqjJVjRZMEvs1PNiFqtlsYKrLMUqS8F0WtMt11nDphNNqU6vNcvlOqepA1EvolKhzFrUApWCOTJzNFejZODBebaaRaZQMrJSKhUrKpVSKUXFqS5uDcxArazLiXV5peIlxay0/RIbZsDdMLVkQorfV8lIpFEmJVW2uB2MdspZnCyDT/lzJXAJnDE+9cy4iqfA4yi8GHKHo9pcsBhY4/XMYm0WzwnecMvKmxUIxXODm3BsFCcPZ0jRJThB+FRaOD94fCx+X6Vokjo+4l6x6viIi6XQLhYjLpaCtUz6w07PMY8nnebHg1FG19D188WruXDeOFfb2tpqHL8Bv6NbtA==";
+
+    public String decode() throws UnsupportedEncodingException, DataFormatException {
+        Decoder decoder = new Decoder();
+        String decodedString = decoder.decode(blueprint);
+        return decodedString;
     }
 
-    public static void main(String[] args) {
-        System.out.println(new App().getGreeting());
+    public static void main(String[] args) throws UnsupportedEncodingException, DataFormatException {
```

Comment:
remove throws if you successfully implemented the try catch in Decoder.decode() Method